packages/web-components/@groupby/elements-sayt/test/unit/sayt.test.ts

```diff
@@ -1108,6 +1108,7 @@ describe('Sayt Component', () => {
 
     it('should call the requestUpdateSearchTerm method on autocomplete when enter is pressed and the event target is within the searchbox component', () => {
       stub(sayt, 'nodeInSearchbox').returns(true);
+      sayt.visible = true;
```

Comment:
Nitpick, but I would put assignments first before function calls. Same goes for the other instances introduced below.